### PR TITLE
fix: exclude the AppleTV devices from the device discovery

### PR DESCRIPTION
**Pull request by DimitarTachev**
Related to: https://github.com/NativeScript/nativescript-cli/issues/5030

### CODE REVIEW

File: IOSDeviceLib/IOSDeviceLib.cpp [hunk 1]
```diff
@@ -239,7 +239,8 @@ void get_device_properties(std::string device_identifier, json &result)
 	result["productType"] = get_device_property_value(device_identifier, "ProductType");
 	result["deviceName"] = get_device_property_value(device_identifier, "DeviceName");
 	result["productVersion"] = get_device_property_value(device_identifier, kProductVersion);
-	result["deviceColor"] = get_device_property_value(device_identifier, "DeviceColor");
+    result["deviceColor"] = get_device_property_value(device_identifier, "DeviceColor");
+    result["deviceClass"] = get_device_property_value(device_identifier, "DeviceClass");
```
Comment (KristianDD) [inline IOSDeviceLib/IOSDeviceLib.cpp]: Changed indentation.

Comment (Fatme) [reply IOSDeviceLib/IOSDeviceLib.cpp]: 😸

File: IOSDeviceLib/IOSDeviceLib.cpp [hunk 2]
```diff
@@ -286,13 +287,17 @@ void on_device_found(const DevicePointer* device_ptr, std::string device_identif
...
+        }
 	}
 }
 
 void device_notification_callback(const DevicePointer* device_ptr)
 {
 	std::string device_identifier = get_cstring_from_cfstring(AMDeviceCopyDeviceIdentifier(device_ptr->device_info));
-	json result;
+	json result = nullptr;
 	result[kDeviceId] = device_identifier;
```
Comment (KristianDD) [inline IOSDeviceLib/IOSDeviceLib.cpp:301]: Not sure this is correct.

Comment (DimitarTachev) [reply IOSDeviceLib/IOSDeviceLib.cpp:301]: Me too, but it's working. It should be the same. The updated version is more clear, as we are comparing with `nullptr` below.

**Review by Fatme — APPROVED** [review]

**Pull request merged**